src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -919,6 +923,28 @@ internal virtual void ManagedOnUnloaded()
 			);
 		}
 
+		/// <summary>
+		/// If corresponding feature flag is enabled, set layout properties as DOM attributes to aid in debugging.
+		/// </summary>
+		private protected virtual void UpdateDOMProperties()
+		{
+			if (FeatureConfiguration.UIElement.AssignDOMXamlProperties)
+			{
+				UpdateDOMXamlProperty(nameof(Visibility), Visibility);
+				UpdateDOMXamlProperty(nameof(IsHitTestVisible), IsHitTestVisible);
+			}
+		}
+
+		/// <summary>
+		/// Sets a Xaml property as a DOM attribute for debugging.
+		/// </summary>
+		/// <param name="propertyName">The property's name</param>
+		/// <param name="value">The current property value</param>
+		private protected void UpdateDOMXamlProperty(string propertyName, object value)
```

Comment:
Why did you renamed it to "Update" instead of "Set"?

Reply:
Because... it's being called from `UpdateDOMProperties()`... I guess...